### DISCUSSION TITLE
HTML to Image

### DISCUSSION THREAD

**Discussion by TeslaSP2** [dlemstra/Magick.NET#1378] [Help]
Can I use this library to make an image from Raw HTML?

**Comment by TeslaSP2** [comment]
Also forgot to say this and now I can't edit the question, I need this to work on a linux Docker, I know a few ways to do it with other libraries on Windows but the styles doesn't apply there for some reason

**Comment by dlemstra** [comment]
There is no support for reading a `HTML` file in the @ImageMagick library and we will not add that in the future. And I also don't know what you could use for this.
